client/lib/plans/constants.js

```diff
@@ -287,79 +313,154 @@ export const PLANS_LIST = {
+			? [
+				FEATURE_CORE_JETPACK,
+				FEATURE_SECURITY_ESSENTIALS_JETPACK,
+				FEATURE_PRIORITY_SUPPORT_JETPACK
+			]
+			: [
+				FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+				FEATURE_BACKUP_ARCHIVE_30,
+				FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED,
+				FEATURE_AUTOMATED_RESTORES,
+				FEATURE_SPAM_AKISMET_PLUS,
+				FEATURE_EASY_SITE_MIGRATION,
+				FEATURE_PREMIUM_SUPPORT
+			],
+		getBillingTimeFrame: () => i18n.translate( 'per month, billed yearly' )
+	},
+
 	[ PLAN_JETPACK_PERSONAL_MONTHLY ]: {
 		getTitle: () => i18n.translate( 'Personal' ),
+		getStoreSlug: () => PLAN_JETPACK_PERSONAL_MONTHLY,
 		getProductId: () => 2006,
 		getPathSlug: () => 'jetpack-personal-monthly',
 		availableFor: ( plan ) => includes( [ PLAN_JETPACK_FREE ], plan ),
-		getDescription: () => i18n.translate(
-			'Essentials for every site. The most affordable solution to keep your personal' +
-			' or small business site backed up and spam-free.'
+		getDescription: ( abtest ) => abtest && abtest( 'jetpackPlansTabs' ) === 'tabs'
+			? i18n.translate( 'Everything you need to get started' )
+			: i18n.translate(
+				'Essentials for every site. The most affordable solution to keep your' +
+				' personal or small business site backed up and spam-free.'
 		),
-		getFeatures: () => [
-			FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
-			FEATURE_BACKUP_ARCHIVE_30,
-			FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED,
-			FEATURE_AUTOMATED_RESTORES,
-			FEATURE_SPAM_AKISMET_PLUS,
-			FEATURE_EASY_SITE_MIGRATION,
-			FEATURE_PREMIUM_SUPPORT
-		],
+		getFeatures: ( abtest ) => abtest && abtest( 'jetpackPlansTabs' ) === 'tabs'
+			? [
+				FEATURE_CORE_JETPACK,
+				FEATURE_SECURITY_ESSENTIALS_JETPACK,
+				FEATURE_PRIORITY_SUPPORT_JETPACK
+			]
+			: [
+				FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+				FEATURE_BACKUP_ARCHIVE_30,
+				FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED,
+				FEATURE_AUTOMATED_RESTORES,
+				FEATURE_SPAM_AKISMET_PLUS,
+				FEATURE_EASY_SITE_MIGRATION,
+				FEATURE_PREMIUM_SUPPORT
+			],
 		getBillingTimeFrame: () => i18n.translate( 'per month, billed monthly' )
 	},
 
 	[ PLAN_JETPACK_BUSINESS ]: {
 		getTitle: () => i18n.translate( 'Professional' ),
 		getProductId: () => 2001,
-		availableFor: ( plan ) => includes( [ PLAN_JETPACK_FREE, PLAN_JETPACK_PREMIUM, PLAN_JETPACK_PREMIUM_MONTHLY, PLAN_JETPACK_PERSONAL, PLAN_JETPACK_PERSONAL_MONTHLY ], plan ),
+		availableFor: ( plan ) => includes( [
+			PLAN_JETPACK_FREE,
+			PLAN_JETPACK_PREMIUM,
+			PLAN_JETPACK_PREMIUM_MONTHLY,
+			PLAN_JETPACK_PERSONAL,
+			PLAN_JETPACK_PERSONAL_MONTHLY
+		], plan ),
 		getPathSlug: () => 'professional',
-		getDescription: () => i18n.translate(
-			'Real-time backups, unlimited archives, and one-click threat ' +
-			'resolution. Also includes SEO tools, and unlimited video hosting.'
+		getDescription: ( abtest ) => abtest && abtest( 'jetpackPlansTabs' ) === 'tabs'
+			? i18n.translate( 'Ultimate security and traffic tools.' )
```

Comment:
Alternate string suggestion: 
* `translate( 'More powerful security tools and realtime content backup for the ultimate peace of mind. Supports up to 3 WordPress sites.' )`&mdash; translations: **18**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-plans-tab)